cmd/helm/helm.go

```diff
@@ -158,7 +158,12 @@ func init() {
 func main() {
 	cmd := newRootCmd(os.Args[1:])
 	if err := cmd.Execute(); err != nil {
-		os.Exit(1)
+		switch e := err.(type) {
```

Comment:
This should be an idiomatic way for this kind of switch as explained in https://github.com/pkg/errors/issues/73#issuecomment-244641060